src/main/java/com/timekeeper/app/dto/Payroll.java

```diff
@@ -9,8 +9,8 @@
 public @Data class Payroll {
     private Employee employee;
     private String date;
-    private double hoursWorked;
-    private double amountPaid;
+    private int hoursWorked;
+    private int amountPaid;
```

Comment:
I changed the the hoursWorked & amountpaid to int. I feel the program would run better

Reply:
Could you explain your reasoning behind this? I think the double is fine for both variable types, as there's reasonable cases for both that would need decimals.

Reply:
I agree with @vumd-UC ... This suggestion is a functionality change, and does not offer benefit in this case.  Only change from double to int if you are certain that there will only be whole numbers.  hoursWorked and amountPaid could easily be fractional.

What I do recommend... double is subject to floating point arithmetic, which can cause inaccuracies when dealing with time and money.  Consider using a BigDecimal instead.